src/Symfony/Component/Notifier/Bridge/Expo/ExpoTransport.php

```diff
@@ -93,6 +93,11 @@ protected function doSend(MessageInterface $message): SentMessage
 
         $success = $response->toArray(false);
 
+        $isError = 'error' === $success['data']['status'];
+        if ($isError) {
```

Comment:
`if ('error' === $success['data']['status']) {`

but let's rename `$success` to `$result`?

Reply:
Yes & Done :)